EIPS/eip-2935.md

```diff
@@ -61,8 +61,8 @@ The history contract has two operations: `get` and `set`. The `set` operation is
 It is used from the EVM for looking up block hashes.
 
 * Callers provide the block number they are querying in a big-endian encoding.
-* If calldata is bigger than 2^64-1, revert.
-* For any output outside the range of [block.number-`HISTORY_SERVE_WINDOW`, block.number-1] return 0.
+* If calldata is not 32 bytes, revert.
+* For any request outside the range of [block.number-`HISTORY_SERVE_WINDOW`, block.number-1], revert.
```

Comment:
this is not the current BLOCKHASH (with window 256) behavior so we are actually diverting from the behavior ?